src/components/FlowRunTimelineOptions.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <p-pop-over
```

Comment:
This is almost identical to `PIconButtonMenu`. I think you can use that component here and just pass in the attrs for your button.

Reply:
You might be right with some additional props on that component. A couple things it came with I didn't want, namely closing the menu on any click within.